### PR TITLE
Minor fix in BLEU

### LINKED ISSUE #90: Integrate bleu_tool.py into texar.evals
related issue https://github.com/asyml/texar-pytorch/issues/88

### PR DESCRIPTION
**Pull request by gpengzhi**
Resolve comments in https://github.com/asyml/texar-pytorch/pull/105

Resolve #90

### CODE REVIEW

File: examples/transformer/bleu_main.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 from argparse import ArgumentParser
 
-from texar.evals.bleu_tool import bleu_wrapper
+from texar.evals.bleu_transformer import bleu_wrapper
```
Comment (ZhitingHu) [inline examples/transformer/bleu_main.py]: Do we still have `bleu_wrapper`?

File: texar/evals/bleu_transformer.py [hunk 2]
```diff
@@ -185,14 +194,18 @@ def bleu_tokenize(string: str) -> List[str]:
...
 
 
-def bleu_wrapper(ref_filename: str,
-                 hyp_filename: str,
-                 case_sensitive: bool = False) -> float:
+def file_bleu(ref_filename: str,
+              hyp_filename: str,
+              bleu_version: str = "corpus_bleu_transformer",
+              case_sensitive: bool = False) -> float:
     r"""Compute BLEU for two files (reference and hypothesis translation).
```
Comment (ZhitingHu) [inline texar/evals/bleu_transformer.py]: If `file_bleu` and `bleu_tokenize` are not specific for the bleu-transformer version, they can be moved to another file (either `bleu.py` or a new file)

Comment (gpengzhi) [reply texar/evals/bleu_transformer.py]: I prefer to keep these two functions in `bleu_transformer` for now. Currently `bleu_tokenize` is just a special case of the original implementation of the tokenization in `multi-bleu.perl` (This is also mentioned in `doc` of `bleu_tokenize`).  I rename `bleu_tokenize` to `bleu_tokenize_transformer`. After we implement the general `bleu_tokenize`, we could move /delete these two function to the correct folder.

Comment (ZhitingHu) [reply texar/evals/bleu_transformer.py]: Ok. I guess `bleu_transformer_tokenize` could prbably a better name?

**Pull request merged**